src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -26,11 +27,19 @@ interface DotEditPageNavItem {
 })
 export class DotEditPageNavComponent implements OnChanges {
     @Input() pageState: DotRenderedPageState;
+    @Output() buttonClicked: EventEmitter<{ inode: string; label: string }> = new EventEmitter();
     model: Observable<DotEditPageNavItem[]>;
     isEnterpriseLicense: boolean;
 
     constructor(private dotLicenseService: DotLicenseService, public dotMessageService: DotMessageService, public route: ActivatedRoute) {}
 
+    emitClick($event: MouseEvent, inode: string, label: string): void {
+        $event.stopPropagation();
+        if (inode) {
+            this.buttonClicked.emit({ inode, label });
```

Comment:
I not sure if is good idea add the 'label' attribute here, we need a better way to identify which button was clicked